src/server.js

```diff
@@ -551,23 +563,6 @@ module.exports = function (classes) {
         type = type.toLowerCase();
         if (type && (_.values(Authorization).indexOf(type) > -1)) {
           this.authType = type;
```

Comment:
we can't rely on the `authType` that is instance wide, need to check as per need using an argument in _checkAuth